src/components/organisms/NavBarSchedule/NavBarSchedule.scss

```diff
@@ -110,7 +80,7 @@ $schedule-btn--height: 51px;
       pointer-events: none;
       cursor: default;
 
-      & .NavBarSchedule__weekday-button {
+      & .NavBarSchedule__weekdayButton {
```

Comment:
Same like above, also in usages in the .tsx file
```suggestion
      & .NavBarSchedule__weekday-button {
```

Reply:
fixed in 2c16847b0928ba957a737453f7b445db887dda23